RecoTauTag/RecoTau/src/AntiElectronIDMVA6.cc

```diff
@@ -886,6 +886,329 @@ double AntiElectronIDMVA6::MVAValue(const reco::PFTau& thePFTau)
+    TauLeadPFChargedHadrEoP = theTau.ecalEnergyLeadChargedHadrCand()/theTau.leadChargedHadrCand()->p();
+  }
+
+  std::vector<Float_t> GammasdEtaInSigCone;
+  std::vector<Float_t> GammasdPhiInSigCone;
+  std::vector<Float_t> GammasPtInSigCone;
+  std::vector<Float_t> GammasdEtaOutSigCone;
+  std::vector<Float_t> GammasdPhiOutSigCone;
+  std::vector<Float_t> GammasPtOutSigCone;
+  reco::Candidate::LorentzVector pfGammaSum(0,0,0,0);
+  reco::Candidate::LorentzVector pfChargedSum(0,0,0,0);
+  
+  const reco::CandidatePtrVector signalGammaCands = theTau.signalGammaCands();
+    for ( reco::CandidatePtrVector::const_iterator gamma = signalGammaCands.begin();
+	  gamma != signalGammaCands.end(); ++gamma ) {
+
+    float dR = deltaR((*gamma)->p4(), theTau.leadChargedHadrCand()->p4());
+    float signalrad = std::max(0.05, std::min(0.10, 3.0/std::max(1.0, theTau.pt())));
+
+    // pfGammas inside the tau signal cone
+    if (dR < signalrad) {
+      if ( theTau.leadChargedHadrCand().isNonnull() ) {
+        GammasdEtaInSigCone.push_back((*gamma)->eta() - theTau.leadChargedHadrCand()->eta());
+        GammasdPhiInSigCone.push_back((*gamma)->phi() - theTau.leadChargedHadrCand()->phi());
+      }
+      else {
+        GammasdEtaInSigCone.push_back((*gamma)->eta() - theTau.eta());
+        GammasdPhiInSigCone.push_back((*gamma)->phi() - theTau.phi());
+      }
+      GammasPtInSigCone.push_back((*gamma)->pt());
+      pfGammaSum += (*gamma)->p4();
+    }
+    // pfGammas outside the tau signal cone
+    else {
+      if ( theTau.leadChargedHadrCand().isNonnull() ) {
+        GammasdEtaOutSigCone.push_back((*gamma)->eta() - theTau.leadChargedHadrCand()->eta());
+        GammasdPhiOutSigCone.push_back((*gamma)->phi() - theTau.leadChargedHadrCand()->phi());
+      } 
+      else {
+        GammasdEtaOutSigCone.push_back((*gamma)->eta() - theTau.eta());
+        GammasdPhiOutSigCone.push_back((*gamma)->phi() - theTau.phi());
+      }
+      GammasPtOutSigCone.push_back((*gamma)->pt());
+    }
+  }
+  
+  const reco::CandidatePtrVector signalChargedCands = theTau.signalChargedHadrCands();
+    for ( reco::CandidatePtrVector::const_iterator charged  = signalChargedCands.begin();
+	  charged  != signalChargedCands.end(); ++charged ) {
+    
+    float dR = deltaR((*charged)->p4(), theTau.leadChargedHadrCand()->p4());
+    float signalrad = std::max(0.05, std::min(0.10, 3.0/std::max(1.0, theTau.pt())));
+  
+    // charged particles inside the tau signal cone
+    if (dR < signalrad) {
+        pfChargedSum += (*charged)->p4();
+    }
+  }
+  
+  Int_t TauSignalPFGammaCandsIn = GammasPtInSigCone.size();
+  Int_t TauSignalPFGammaCandsOut = GammasPtOutSigCone.size();
+  Float_t TauVisMassIn = (pfGammaSum + pfChargedSum).mass();
+
+  Float_t TauPhi = theTau.phi();
+ 
+  if ( theTau.phiAtEcalEntrance() != -99. ) {
+    TauPhi = theTau.phiAtEcalEntrance();
+  }
+  Float_t TaudCrackPhi = dCrackPhi(TauPhi, TauEtaAtEcalEntrance);
```

Comment:
Can you replace this one with something that is available on or can be obtained from miniAOD instead of the missing variable? I'll use this version than for a PR that does not rely on the update of the miniAOD event content. Once we get the update on the miniAOD event content integrated (or find another solution to the issue), we can update this section of the code correspondingly. But by that time the bulk of the commit could already be in review or even integrated.